### PR TITLE
fix: issue template updates for Figma link

### LINKED ISSUE #128: Issue Template Updates for Figma link
#### Current Behavior
The project's current issue templates are missing an open invitation link where new contributors can join the community's Figma team and view user interface designs and other UX projects.

#### Desired Behavior
Each template that has a reference to Figma in its resources section should an invite link added.

#### Implementation

In each [issue template](../blob/master/.github/ISSUE_TEMPLATE/) that refers to Figma, replace the current line referring to Figma with this line:

```
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
#### Acceptance Tests
1. All references to Figma include the "open invite" link.
---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by depleur**
**Notes for Reviewers**

This PR fixes #128

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -34,5 +34,5 @@ assignees: ''
 - 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
 - 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
    - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
-- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]:
```suggestion
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -17,5 +17,5 @@ assignees: ''
 - 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
    - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
 - 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
-- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/documentation.md:20]:
```suggestion
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Removed unwanted backticks.

@depleur thanks. Next time, please adhere to the DCO requirement.

**Comment by depleur** [comment]
> Removed unwanted backticks.
> 
> @depleur thanks. Next time, please adhere to the DCO requirement.

Hey, thanks. I just travelled to my village where I can't really do much. Thanks for all the help and I'll stick to the DCO stuff henceforth.

**Pull request merged**